infra/cmake/modules/IdentifyPlatform.cmake

```diff
@@ -49,6 +51,8 @@ endif()
 
 if("${TARGET_ARCH}" STREQUAL "x86_64")
   set(TARGET_ARCH_BASE ${TARGET_ARCH})
+elseif("${TARGET_ARCH}" STREQUAL "armv7em")
+  set(TARGET_ARCH_BASE "arm")
```

Comment:
@hseok-oh, I introduced armv7em arch, according to https://github.com/Samsung/ONE/pull/9477#issuecomment-1193400713 comment